src/filters/fs.rs

```diff
@@ -106,8 +106,7 @@ fn path_from_tail(
         .and_then(move |tail: crate::path::Tail| {
             future::ready(sanitize_path(base.as_ref(), tail.as_str()))
                 .and_then(|mut buf| async {
-                    let clone_buf = buf.clone();
-                    let is_dir = tokio_executor::blocking::run(move || clone_buf.is_dir()).await;
+                    let is_dir = tokio::task::block_in_place(|| buf.is_dir());
```

Comment:
I think this *requires* the multi-threaded runtime to be running, whereas it could be written like this:

```rust
let is_dir = tokio::fs::metadata(buf).await.map(|m| m.is_dir()).unwrap_or(false);
```

Reply:
sorry, it was me who suggested `block_in_place` in https://github.com/seanmonstar/warp/pull/336#discussion_r356627991 as i saw fs filter already requires the multi threaded runtime

Reply:
It definitely *did* in 0.1, but tokio 0.2 can now allow fs operations even with the basic scheduler.

Reply:
ah nice! fs filter doc can also be updated then, as it no longer needs to reference that: https://docs.rs/warp/0.1.10/warp/filters/fs/fn.dir.html#note

Reply:
cool! this is what I wanted but didn't know existed.